### DISCUSSION TITLE
Error in Upload proof of your academic status

### DISCUSSION THREAD

**Discussion by AnthonyRiveraI** [community/community#75795] [GitHub Education]
### Hi everyone,

![image](https://github.com/community/community/assets/148939728/22d36b70-a315-4c74-96b4-2a78390b5ebd)

**Comment by vgoulartdt** [comment]
I'm facing the same problem (if I guess it right, because you do not say anything besides uploading the image), I'm not able to upload my university letter to proof I'm a student, probably because of this same error: {:class=>"discount_request_id", :data=>{:discount_request_id=>12636187}}. It's only allowing me for taking a photo but it's not for uploading a file. What's about taking a photo if I have the document myself on my PC ready to be uploaded?

I really don't know if this is the cause of the problem, but I saw a video on YouTube about subscribing to GitHub Student Developer Pack, in that case it was showing a button to Upload an Image and it was not showing this piece of code which seem to be an error ({:class=>"discount_request_id", :data=>{:discount_request_id=>12636187}}) just like it is happening with us. I'm using Google Chrome, with other browsers I could't go so far...
